saleor/product/models.py

```diff
@@ -299,6 +299,16 @@ def sort_by_attribute(
             f"{ordering}name",
         )
 
+    def prefetched_product_for_webhook(self):
```

Comment:
IMO `prefetched_for_webhook` has enough information. No need to duplicate the `product` word
`Product.objects. prefetched_for_webhook()`